kpi-tracker/src/main.rs

```diff
@@ -209,48 +249,85 @@ fn get_account_transaction_details(
 fn to_block_events(block_hash: BlockHash, block_item: BlockItemSummary) -> Vec<BlockEvent> {
     let mut events: Vec<BlockEvent> = Vec::new();
 
-    match block_item.details {
+    match &block_item.details {
         AccountTransaction(atd) => {
-            let details = get_account_transaction_details(&atd, block_hash);
-            println!(
-                "TRANSACTION:\nhash: {}\ndetails: {:?}",
-                block_item.hash, &details
-            ); // Logger debug
-            let event = BlockEvent::AccountTransaction(block_item.hash, details);
+            let details = get_account_transaction_details(atd, block_hash);
+            let affected_accounts: Vec<TransactionAccountRelation> = block_item
+                .affected_addresses()
+                .into_iter()
+                .map(|address| {
+                    TransactionAccountRelation(
+                        CanonicalAccountAddress::from(address),
+                        block_item.hash,
+                    )
+                })
+                .collect();
+
+            let affected_contracts: Vec<TransactionContractRelation> = block_item
+                .affected_contracts()
+                .into_iter()
+                .map(|address| TransactionContractRelation(address, block_item.hash))
+                .collect();
+
+            log::debug!(
+                "Account transaction found: {}\n{:?}\n{}\n{}",
```

Comment:
Generally we should keep logs on one line. Multi-line logs are hard to use. Emit multiple lines with multiple `debug` statements if it is too much to fit on one line.